tools/docker_dev/.gitpod.Dockerfile

```diff
@@ -10,8 +10,6 @@ FROM ${BASE_CONTAINER}
 RUN apt-get update && \ 
     apt-get install -yq --no-install-recommends \
     bash-completion \
-    build-essential \
-    git \
```

Comment:
ah I see now the the env dependency relnship here between the gitpod and the docker